Fixed Grafana dashboards to work with new metrics

nihadtz opened the pull request:
Dashboards on grafana are not working because old metrics are used. This PR will fix it.

Mostly it was replacement of
_hono_messages_received_seconds_count_ =>
_hono_telemetry_processing_duration_seconds_count_

_hono_messages_payload_bytes_sum_ =>
_hono_telemetry_payload_bytes_sum_

_hono_commands_received_seconds_count_ =>
_hono_command_processing_duration_seconds_count_

_hono_commands_payload_bytes_sum_ =>
_hono_command_payload_bytes_sum_

<img width="2472" alt="Screenshot 2023-03-03 at 08 53 21" src="https://user-images.githubusercontent.com/46897537/222663320-6278e566-c07e-45ef-893a-b629340e5a30.png">
<img width="2493" alt="Screenshot 2023-03-03 at 08 53 41" src="https://user-images.githubusercontent.com/46897537/222663327-99d185e0-2d92-4f04-af82-43e2a24b81c2.png">

calohmn commented:
@nihadtz Thanks for the PR!
In order to contribute, you will need to agree to the Eclipse Contributor Agreement (ECA) as described on our [Contributing page](https://www.eclipse.org/packages/contribute/#the-legal-side-of-things).

> Mostly it was replacement of
hono_messages_received_seconds_count =>
hono_telemetry_payload_bytes_count

Actually, this is a different metric then. With Hono 2.0, the `hono.messages.received.*` metrics got renamed to `hono.telemetry.processing.duration.*` (see also the [release notes](https://www.eclipse.org/hono/release-notes/#200) for Hono 2.0, under "API Changes").

nihadtz commented:
> @nihadtz Thanks for the PR! In order to contribute, you will need to agree to the Eclipse Contributor Agreement (ECA) as described on our [Contributing page](https://www.eclipse.org/packages/contribute/#the-legal-side-of-things).
> 
> > Mostly it was replacement of
> > hono_messages_received_seconds_count =>
> > hono_telemetry_payload_bytes_count
> 
> Actually, this is a different metric then. With Hono 2.0, the `hono.messages.received.*` metrics got renamed to `hono.telemetry.processing.duration.*` (see also the [release notes](https://www.eclipse.org/hono/release-notes/#200) for Hono 2.0, under "API Changes").

Thank you @calohmn i was not aware of the changelogs.

I have updated PR description, replaced right metrics names, squashed commits and forced update on branch.

nihadtz commented:
@calohmn do we need to update chart version to 2.3.1 from 2.3.0 as i see its reporting lint issue?

calohmn commented:
@nihadtz yes, indeed. The chart version has to be incremented with each chart change. 
And it would be good if you could add a short note about what changed in 2.3.1 in the `Release notes` section in `charts/hono/README.md`.

nihadtz commented:
> @nihadtz yes, indeed. The chart version has to be incremented with each chart change. And it would be good if you could add a short note about what changed in 2.3.1 in the `Release notes` section in `charts/hono/README.md`.

I have added release notes and changed version to 2.3.1, not sure if i need to change on both places in `charts/hono/Chart.yaml`

calohmn commented on charts/hono/Chart.yaml:
```diff
@@ -15,9 +15,9 @@ name: hono
+version: 2.3.1
 # Version of Hono being deployed by the chart
-appVersion: 2.3.0
+appVersion: 2.3.1
```
Only `version` (i.e. chart version) needs to be changed here.
`appVersion` refers to the Hono release version to be used - it needs to stay unchanged here.

nihadtz replied:
Thank you, its changed to proper values now.

calohmn reviewed (approved):
LGTM
Thanks for contributing!
